be/src/exprs/time_functions.cpp

```diff
@@ -1943,7 +1943,15 @@ Status TimeFunctions::parse_joda_close(FunctionContext* context, FunctionContext
     return {};
 }
 
+StatusOr<ColumnPtr> TimeFunctions::trino_parse_jodatime(FunctionContext* context, const Columns& columns) {
+    return parse_jodatime_base(context, columns, true);
```

Comment:
It’s best not to flag trino

Reply:
Would it be a better choice to pass the sql_dialect parameter to be？

Reply:
sure

Reply:
I have modified the implementation, could you please review it again?